CONTRIBUTING.md

```diff
@@ -41,240 +37,98 @@ There are several docker compose setups to aid in speeding up the development.
   - The **docker-compose.yml** file contains the most likely setup with development setups for both the api and web projets. This should be used when working on both the api and the web front end. You can run this using **run-api-web.sh**.
   - The **docker-compose.full.yml** contains a setup with all of the containers needed to run Doubtfire with overseer. This requires access to the overseer repository. You can run this using **run-full.sh**
 
-### Working with Docker Compose
-
-To get started:
-
-1. Fork [doubtfire-deploy](https://github.com/doubtfire-lms/doubtfire-deploy), [doubtfire-api](https://github.com/doubtfire-lms/doubtfire-api), and [doubtfire-web](https://github.com/doubtfire-lms/doubtfire-web)
-
-    To push your contributions, you will need a fork of each repository. Contributions can then be made by making pull requests back into the main repositories.
-
-2. Clone your [doubtfire-deploy](https://github.com/doubtfire-lms/doubtfire-deploy). Make sure to fetch submodules to get the subprojects.
+## Getting Started
 
-    `git clone --recurse-submodules https://github.com/YOUR_USERNAME/doubtfire-deploy`
+To run OnTrack locally on your machine:
 
-3. Open a Terminal that supports `sh` scripts (on Windows, you will need WSL, Msys2, or Cygwin). Run the following command to set your fork as the remote.
+1. Open a terminal and clone Thoth Tech's doubtfire-deploy repo
 
-    `./change_remotes.sh`
+    `git clone --recurse-submodules https://github.com/thoth-tech/doubtfire-deploy`
```

Comment:
**nitpick:** Might be better to turn it into a code block instead of inline?

```suggestion
    ```sh
    git clone --recurse-submodules https://github.com/thoth-tech/doubtfire-deploy
    ```
```